adapter.go

```diff
@@ -1,19 +1,22 @@
 package adapters
 
 import (
+	"database/sql"
 	"net/http"
 	"net/url"
 )
 
 //Adapter interface
 type Adapter interface {
+	GetTx() (tx *sql.Tx, err error)
```

Comment:
I prefer explicit name, example `GetTransaction`

Reply:
+1 to GetTransaction